vscode/quint-vscode/server/package.json

```
@@ -12,7 +12,7 @@
 		"url": "https://github.com/informalsystems/quint"
 	},
 	"dependencies": {
-        "@informalsystems/quint": "^0.4.0",
+		"@informalsystems/quint": "file:.yalc/@informalsystems/quint",
```

This is a local change made by `make local` that is not supposed to be commited.

```suggestion
		"@informalsystems/quint": "^0.4.0",
```